src/System.Management.Automation/DscSupport/JsonDeserializer.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+
+namespace Microsoft.PowerShell.DesiredStateConfiguration.Json
+{
+    internal class JsonDeserializer
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Instantiates a default deserializer.
+        /// </summary>
+        public static JsonDeserializer Create()
+        {
+            return new JsonDeserializer();
+        }
+
+        #endregion Constructors
+
+        #region Methods
+
+        /// <summary>
+        /// Returns schema of Cim classes from specified json file.
+        /// </summary>
+        public IEnumerable<PSObject> DeserializeClasses(string json, bool useNewRunspace = false)
+        {
+            if (string.IsNullOrEmpty(json))
+            {
+                throw new ArgumentNullException(nameof(json));
+            }
+
+            IEnumerable<PSObject> result = null;
+            System.Management.Automation.PowerShell powerShell = null;
```

Comment:
You already have `using System.Management.Automation` above, but notice you specify the namespace multiple times in this file?

Reply:
without full namespace I'm getting build errors like:

- JsonDeserializer.cs(39,13): error CS0118: 'PowerShell' is a namespace but is used like a type
- JsonDeserializer.cs(51,30): error CS0234: The type or namespace name 'Create' does not exist in the namespace 'Microsoft.PowerShell' (are you missing an assembly reference?)

Reply:
You can fix it by putting the `using` within the `namespace` scope